src/test/scala/tests/shared/IcebergCatalogInfo.scala

```diff
@@ -0,0 +1,19 @@
+package com.sneaksanddata.arcane.framework
+package tests.shared
+
+import services.lakehouse.IcebergCatalogCredential
+import services.lakehouse.base.{IcebergCatalogSettings, S3CatalogFileIO}
+
+object IcebergCatalogInfo:
+  val defaultNamespace = "test"
+  val defaultWarehouse = "demo"
+  val defaultCatalogUri = "http://localhost:20001/catalog"
+  
+  val defaultSettings: IcebergCatalogSettings = new IcebergCatalogSettings:
+    override val namespace: String = defaultNamespace
+    override val warehouse: String = defaultWarehouse
+    override val catalogUri: String = defaultCatalogUri
+    override val additionalProperties: Map[String, String] = IcebergCatalogCredential.oAuth2Properties
```

Comment:
Do we still need auth properties in unit tests? I thought, we don't use OAuth in the docker-compose for unit tests?

Reply:
Created https://github.com/SneaksAndData/arcane-framework-scala/issues/143 to take a look at this